cloudmarker/alerts/emailalert.py

```diff
@@ -12,98 +12,86 @@
-            subject (str): Subject line for email.
-            to (list): List of recipients (str).
-            sender (str): From field for email.
-            body (str): Email body.
-            use_ssl (bool): Is SSL connection required
-            username (str): username of email account
-            password (str): password of email account
+            from_addr (str): Sender's email address.
+            to_addrs (list): A list of :obj:`str` objects where each
+                :obj:`str` object is a recipient's email address.
+            host (str): SMTP host.
+            port (int): SMTP port.
+            use_ssl (bool): Use SSL if ``True``, not otherwise.
+            username (str): SMTP username.
+            password (str): SMTP password.
         """
         self._host = host
         self._port = port
         self._subject = subject
-        self._to = to
-        self._sender = sender
-        self._body = body
+        self._from_addr = from_addr
+        self._to_addrs = to_addrs
         self._stringbuffer = []
         self._use_ssl = use_ssl
         self._username = username
         self._password = password
 
     def write(self, record):
-        """Write JSON records to the file system.
-
-        This method is called once for every ``record`` read from a
-        cloud. In this example implementation of a alert, we simply
-        send the ``record`` in JSON format via email to the recipient.
-        The records keeps appending to a stringbuffer which then is
-        written in the email body
+        """Save event record in a buffer.
 
         Arguments:
-            record (dict): Data to send via email.
+            record (dict): An event record.
         """
         for _, value in record.items():
             self._stringbuffer.append(repr(value))
 
     def done(self):
-        """Perform final cleanup tasks.
-
-        This method is called after all records have been written. In
-        this example implementation, we properly terminate the JSON
-        array in the email body.
-
-        The connection is made based whether the SMTP or SMTP_SSL is
-        required which is determined by ``use_ssl`` param. The SMTP
-        login process is conducted if ``username`` param is provided.
-
-        """
+        """Send the buffered events as an email alert."""
         message = MIMEMultipart()
         message['Date'] = email.utils.formatdate(localtime=True)
         message['Subject'] = self._subject
-        message['From'] = self._sender
-        message['To'] = email.utils.COMMASPACE.join(self._to)
+        message['From'] = self._from_addr
+        message['To'] = email.utils.COMMASPACE.join(self._to_addrs)
 
-        # Incase if the buffer is empty then the default message will be sent
-        if self._stringbuffer:
-            self._body = ''.join(self._stringbuffer)
-        message.attach(MIMEText(self._body))
+        body = ''.join(self._stringbuffer)
```

Comment:
Is it okay to send raw json data in the email body?

Reply:
It is not okay to send raw JSON in the email body. I would like `record['com']['description']` and `record['com']['recommendation']` to be picked out from the event record (`record`) and formatted neatly in the email alerts. This will be done in a separate pull request.